packages/cli/src/commands/generate/generator/generator.js

```diff
@@ -0,0 +1,97 @@
+import path from 'path'
+
+import fse from 'fs-extra'
+import Listr from 'listr'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from '../../../lib'
+import c from '../../../lib/colors'
+
+export const command = 'generator <name>'
+export const description =
+  'Copies generator templates locally for customization'
+
+const SIDE_MAP = {
+  web: ['cell', 'component', 'layout', 'page', 'scaffold'],
+  api: ['function', 'sdl', 'service'],
+}
+
+const copyGenerator = (name, { force }) => {
+  const side = SIDE_MAP['web'].includes(name) ? 'web' : 'api'
+  const from = path.join(__dirname, '..', name, 'templates')
+  const to = path.join(getPaths()[side].generators, name)
+
+  // copy template files to appropriate side in app
+  fse.copySync(from, to, { overwrite: force, errorOnExist: true })
+
+  return to
+}
+
+// This could be built using createYargsForComponentGeneration;
+// however, functions wouldn't have a `stories` option. createYargs...
+// should be reversed to provide `yargsDefaults` as the default configuration
+// and accept a configuration such as its CURRENT default to append onto a command.
+export const builder = (yargs) => {
+  yargs
+    .positional('name', {
+      description: 'Name of the generator to copy templates from',
+      choices: [
```

Comment:
Crazy idea here @cannikin - what if these choices were dynamic? We already have generators organised into the folders by their names...

Reply:
Should *every* generator in that folder be customizable? I left some out for the following reasons: 

* `dataMigration` - nothing to really customize, the whole thing is custom
* `dbAuth` - only ever run one time, to create optional login/signup pages
* `script` - again, nothing to customize really
* `secret` - outputs a randomized string to the console, doesn't create any files

Reply:
Hmmm - we could do a filter I suppose! 

What I like about dynamic choices is that 🤞 we don't need to change this when we add new generators!

Reply:
hahaha I'll bookmark this discussion so I can remind you about this in a couple of months. ;)

Reply:
Although wait, wouldn't the `filter` command now have a hardcoded list of the ones to *exclude* instead? Same problem!

Reply:
Similar but not the same, I think! 

A blacklist in this case (I.e filter) would mean we can remove generators we don't want when we realise they're not really useful. 

A whitelist would mean we would need to actively remember to add new generators - so the impact is worse, IMHO. 

I think I prefer "custom templates for x are not useful" to "cannot generate custom templates for y"

Reply:
I think most (if not all) generators we add in the future will be excluded (because they won't have customizable templates) so we'll end up editing this file every time, but if that's what you wanna dooooooo 😉

Reply:
Updated to use filter and blacklist!